omero/sysadmins/server-upgrade.rst

```diff
@@ -46,6 +46,20 @@ to enable successful upgrading of the database, otherwise the upgrade
 script aborts with a message saying that your database server version is
 less than the OMERO prerequisite.
 
+Corrupted pyramids
+^^^^^^^^^^^^^^^^^^
+
+A bug introduced in OMERO 5.2 meant that corrupted pyramids were generated for
```

Comment:
Maybe mention the full version number so that people can tell if they were affected?